flink-kubernetes/src/main/java/org/apache/flink/kubernetes/highavailability/KubernetesStateHandleStore.java

```diff
@@ -649,6 +649,7 @@ private Optional<KubernetesConfigMap> addEntry(
             KubernetesConfigMap configMap, String key, byte[] serializedStateHandle)
             throws Exception {
         final String content = configMap.getData().get(key);
+        final String newContent = toBase64(serializedStateHandle);
```

Comment:
```suggestion
        final String oldBase64Content = configMap.getData().get(key);
        final String newBase64Content = toBase64(serializedStateHandle);
```
nit: We might want to make the previously used variable also more explicit to differentiate both purposes.